### DISCUSSION TITLE
Manually generating maps

### DISCUSSION THREAD

**Discussion by csidirop** [organicmaps/organicmaps#2986] [Q&A] [answered]
I would like to generate maps manually so I can test and verify osm changes. Do you have instructions how to generate them? I'm sure I missed them somewhere...

**Comment by biodranik** [comment, marked as answer]
https://github.com/endim8/organicmaps/blob/update-maps_generator-readme/tools/python/maps_generator/README.md

**Reply by csidirop** [reply]
Thanks, exactly what I was looking for!
